Mathlib/Data/Real/Sqrt.lean

```diff
@@ -508,3 +528,12 @@ theorem ContinuousOn.sqrt (h : ContinuousOn f s) : ContinuousOn (fun x => sqrt (
 theorem Continuous.sqrt (h : Continuous f) : Continuous fun x => sqrt (f x) :=
   continuous_sqrt.comp h
 #align continuous.sqrt Continuous.sqrt
+
+namespace Finset
+
+lemma cauchy_schwarz_sqrt {α : Type*} (s : Finset α) (f g : α → ℝ) :
```

Comment:
I think this should be `sum_mul_le_sqrt_mul_sqrt` or similar, to match `sum_mul_sq_le_sq_mul_sq`